apis/projectcontour/v1alpha1/contourconfig_helpers.go

```diff
@@ -330,3 +334,17 @@ func endpointsInConfict(health *HealthConfig, metrics *MetricsConfig) error {
 	}
 	return nil
 }
+
+// healthEndpointsInConflict returns an error if the same address and port are used between the overload manager enforced health listener and metrics
+// _or_ the stats listener. Since the metrics would be validated against the stats listener already we only need to check that the overload manager
+// health listener is not in conflict with either metrics or stats
+func healthEndpointsInConflict(metrics *MetricsConfig, stats, omEnforcedHealth *HealthConfig) error {
```

Comment:
nit, just bc when doing a quick read `stats` made me go look and see if there was another listener for other stats that i forgot about, when this was in fact the normal health check listener

```suggestion
func healthEndpointsInConflict(metrics *MetricsConfig, health, omEnforcedHealth *HealthConfig) error {
```

(or another name that is more clear)

Reply:
updated in https://github.com/projectcontour/contour/pull/6794/commits/f3ff00455c158b83e33d1906f976788457d1faa1